code/modules/atmospherics/environmental/LINDA_fire.dm

```diff
@@ -107,7 +107,7 @@
 	if(burn_time)
 		QDEL_IN(src, burn_time)
 
-/proc/fireflash(atom/center, radius, temp)
+/proc/fireflash(atom/center, radius, temp, faker)
```

Comment:
Faker???

(What's this for?)

Reply:
I forgot to remove this, this was me trying to have fireflashes with lower alpha before I figured out a diff solution

Reply:
Removed!